App.tsx

```diff
@@ -85,48 +58,121 @@ class App extends Component<{}, appState> {
       },
     });
   };
-
+  setColorBlindMode = (colorBlindMode: colorBlindMode) => {
+    const newColors = ColorPicker(colorBlindMode)
+        this.setState({
+            colors: newColors,
+            settings:{
+                colorBlindMode: colorBlindMode,
+                showSettingsScreen: false
+            }
+        });
+  }
   render() {
-    const { region, polygons, markers } = this.state;
-
+    const { region, polygons, markers, settings, colors} = this.state;
+    const styles = StyleSheet.create({
```

Comment:
Not to my knowledge,
The style sheets can change with each call of render() because of the color blind mode feature (i.e. colors.mapMarkerColor can refer to a different color string depending on which mode is enabled) so the style sheet would need to be re-rendered with the new color.